libpoolprotocols/PoolManager.cpp

```diff
@@ -415,7 +418,9 @@ void PoolManager::rotateConnect()
     if (!m_Settings.connections.empty() && m_Settings.connections.at(m_activeConnectionIdx)->Host() != "exit")
     {
         if (p_client)
+        {
```

Comment:
Remove this change.

Reply:
fixed in https://github.com/ethereum-mining/ethminer/pull/1824/commits/eb95ad3ff0c00561f54b881424141c031a677da2